src/main/java/com/homework/Client.java

```diff
@@ -0,0 +1,17 @@
+package main.java.com.homework;
+
+public class Client extends Thread {
+    public static Server server = new Server();
```

Comment:
Why Server is static?

Reply:
Because I think, that server must be one for all. 
Not create a new Server for new Client.